src/components/layout/nav.js

```diff
@@ -607,15 +607,26 @@ const NavDesktop = ({ base, is_ppc, is_ppc_redirect }) => {
 }
 
 export const Nav = ({ base, is_ppc_redirect, is_ppc }) => {
+    const [is_logged_in, setLoggedIn] = useState(false)
+
+    useEffect(() => {
+        setLoggedIn(isLoggedIn())
+    }, [])
```

Comment:
Would you want the `useEffect` to watch on the `is_logged_in` state and render accordingly? Or should the behaviour be only on the refresh of the page?

Reply:
atm, i dont think there is a way to watch on cookies atm, let me know if you found smt otherwise